src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/PropertyAccessorSymbolReferenceFinder.cs

```diff
@@ -34,14 +32,64 @@ protected override bool CanFind(IMethodSymbol symbol)
             return result;
         }
 
-        protected override Task<ImmutableArray<Document>> DetermineDocumentsToSearchAsync(IMethodSymbol symbol, Project project, IImmutableSet<Document> documents, CancellationToken cancellationToken)
+        protected override async Task<ImmutableArray<Document>> DetermineDocumentsToSearchAsync(IMethodSymbol symbol, Project project, IImmutableSet<Document> documents, SymbolFinderOptions options, CancellationToken cancellationToken)
         {
-            return FindDocumentsAsync(project, documents, cancellationToken, symbol.Name);
+            var explicitUsages = await FindDocumentsAsync(project, documents, cancellationToken, symbol.Name).ConfigureAwait(false);
+
+            var property = symbol.AssociatedSymbol as IPropertySymbol;
+            if (property == null || !options.IncludeImplicitAccessorUsages)
+            {
+                return explicitUsages;
+            }
+
+            return explicitUsages.Concat(await FindDocumentsAsync(project, documents, cancellationToken, property.Name).ConfigureAwait(false));
         }
 
-        protected override Task<ImmutableArray<ReferenceLocation>> FindReferencesInDocumentAsync(IMethodSymbol symbol, Document document, CancellationToken cancellationToken)
+        protected override async Task<ImmutableArray<ReferenceLocation>> FindReferencesInDocumentAsync(IMethodSymbol symbol, Document document, SymbolFinderOptions options, CancellationToken cancellationToken)
         {
-            return FindReferencesInDocumentUsingSymbolNameAsync(symbol, document, cancellationToken);
+            var explicitUsages = await FindReferencesInDocumentUsingSymbolNameAsync(symbol, document, cancellationToken).ConfigureAwait(false);
+
+            var property = symbol.AssociatedSymbol as IPropertySymbol;
```

Comment:
events?

Reply:
Wouldn't be handled in this class. See TODO.

Reply:
Great.  when you do that, add comments accordingly.  simple enogh to just say: "event accessors are handle in XXX".  Thanks!